app/src/main/java/com/example/lunchtray/LunchTrayScreen.kt

```diff
@@ -107,16 +109,16 @@ fun LunchTrayApp() {
         NavHost(
             navController = navController,
             startDestination = LunchTrayScreen.Start.name,
-            modifier = Modifier.padding(innerPadding),
         ) {
             composable(route = LunchTrayScreen.Start.name) {
                 StartOrderScreen(
                     onStartOrderButtonClicked = {
                         navController.navigate(LunchTrayScreen.Entree.name)
                     },
                     modifier = Modifier
-                        .padding(dimensionResource(R.dimen.padding_medium))
                         .fillMaxSize()
+                        .padding(innerPadding)
```

Comment:
Combine the two `.padding()`  function calls into 1 or remove 1 depending on what you need.